arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'How to configure an Orbit chain using the Orbit SDK'
+sidebar_label: 'Orbit Chain Configuration'
+description: 'How to configure an Orbit chain using the Orbit SDK '
+author: Mehdi Salehi
+sme: Mehdi Salehi
+target_audience: 'Developers deploying and maintaining Orbit chains.'
+sidebar_position: 6
+---
+
+This guide will explore the essentials of configuring your Orbit chain. Configuration includes a range of settings from the parent chain and node configuration to specific child chain parameter configurations. Our focus here is on child chain configuration, guiding you through the necessary adjustments to customize your chain's operation.
```

Comment:
```suggestion
In this how-to, you'll learn how to configure your Orbit chain using the Orbit SDK. This includes **parent chain**, **node**, and **child chain parameter** configuration.
```

- nit - "introduce the document as its content type explicitly" (undocumented pattern, added to pattern backlog [here](https://www.notion.so/arbitrum/Pattern-guide-Docs-dafaf8d1ce34433d88c1ef3b4b99c19c?pvs=4#a815de55df444b98ba8ac990424a1353))
- nit - consistent section labeling between this intro, and the header titles down below